src/common/gui/CScalableBitmap.cpp

```diff
@@ -3,6 +3,15 @@
 #include <CoreFoundation/CoreFoundation.h>
 #endif
 
+/*
+** This is not the user zoom factor; this is the all in user + display zoom factor
+*/
```

Comment:
Not sure if I understand this comment and maybe it should be with `//`? A very basic question: what are user and display zoom factors and what is this global variable?

Reply:
User zoom factors are things like "1 want this twice as large"
Display zoom factors are things like "My display has 4 physical pixels per logical pixel" (so like retina displays)

This is a static because I don't have access to every instance of a bitmap; it's a global state of the world. But it is a global because I was dumb and should have made it a private static member of CScalableBitmap which I'll do.